sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_interface_loader.cpp

```diff
@@ -31,8 +31,6 @@ UR_DLLEXPORT ur_result_t UR_APICALL urGetGlobalProcAddrTable(
     return retVal;
   }
 
-  pDdiTable->pfnInit = urInit;
```

Comment:
we dont need anymore urInit and urTearDown? here we are removing it from all the tables, so are we removing them from the spec?

Reply:
These changes look correct to me, `urInit` and `urTearDown` are now loader-only so should no longer be implemented by the individual adapters.

That said our last attempt to bump the UR commit was reverted in https://github.com/intel/llvm/pull/11227 so I'm not sure if we can bump it again until it's clear whatever issues were occurring are resolved.

Reply:
> @aelovikov-intel, have these CI issues since been resolved? This PR blocks a lot of other work since it's not possible to add new PI APIs without upgrading UR.

No. Every time I watching the PRs created in my timezone everything goes smoothly. Then overnight/weekend I see our runners in a bad state. I'll repeat - please do NOT create/update any changes for the UR update until we get at least two days with everything green in CI.

Reply:
I believe the issues have been resolved with https://github.com/intel/llvm/pull/11306. Please merge latest origin/sycl for re-test.